pkgs/development/compilers/zig/0.13/default.nix

```diff
@@ -73,6 +73,7 @@ stdenv.mkDerivation (finalAttrs: {
 
     for file in ../doc/langref/*.zig; do
       stage3/bin/zig run ../tools/doctest.zig -- --zig $PWD/stage3/bin/zig --cache-root cache-root -i "$file" -o "codedir/$(basename "$file").out"
+      cp "$file" codedir/
```

Comment:
1. Why this copy? Can't this file be generated already inside codedir in the previous command?
2. this should be squashed with the other commit.

Reply:
1. Not sure, I've been trying to get Zig working with little success
2. I thought I squashed it but must've forgot

Reply:
Looking through the errors on Ofborg, it seems I mixed up the docgen needing a file called `hello.out` but I thought it was `hello.zig`. I made an adjustment with how `doctest` is called so this should work now. I also squashed the commit.

Reply:
Got as far as `./doc/langref.html.in:2878:14: error: unable to open 'handle_error_with_catch_block.zig.out': FileNotFound`. I think this next build I try might work.